apps/events/tests/test_static_trigger.py

```
@@ -74,3 +77,21 @@ def test_last_timeout_can_end_conversation(session):
     session.refresh_from_db()
     assert session.ended_at is not None
     assert static_trigger.event_logs.count() == 1
+
+
+@pytest.mark.django_db()
```

Ruff

Use `@pytest.mark.django_db` over `@pytest.mark.django_db()`

Remove parentheses

(PT023)